FirebaseDatabase/Sources/Utilities/FNextPushId.h

```diff
@@ -20,8 +20,8 @@
 
 + (NSString *)get:(NSTimeInterval)now;
 
-+ (NSString *)successor:(NSString *)key;
++ (NSString *)from:(NSString *)fn successor:(NSString *)key;
```

Comment:
Right, I was trying to follow the precedent.

I'll gladly change it, but I'm uncertain about wether you're actually asking me to change it based on your preference, or if you're stating that it's probably ok to go with the precedent?

Reply:
I will leave this up to you. There is something to be said about consistency within a repo, so I am okay with either. I just wanted to point this out in case you had similar thoughts.

Reply:
I must admit that I didn't really consider this, I just added the extra arguments rather automatically. I'll just leave it as is for now.